test/unit/event.js

```diff
@@ -2816,6 +2816,28 @@ QUnit.test( "originalEvent property for Chrome, Safari, Fx & Edge of simulated e
 	jQuery( "#donor-input" ).trigger( "focus" );
 } );
 
+QUnit.test( "Event aliases", function( assert ) {
+
+	// Explicitly skipping focus/blur events due to their flakiness
+	var aliases = ("resize scroll click dblclick " +
+		"mousedown mouseup mousemove mouseover mouseout mouseenter mouseleave " +
+		"change select submit keydown keypress keyup contextmenu").split(" ");
+	assert.expect( aliases.length );
+
+	jQuery.each( aliases, function( i, name ) {
+
+		if ( !jQuery.fn[ name ] ) {
+			assert.ok( true, "no method present for jQuery.fn." + name );
+			return;
+		}
+
+		// e.g. $(elem).click(...).click();
+		jQuery( document.body )[ name ]( function( event ) {
```

Comment:
An element under `#qunit-fixture` would probably be better than `document.body`, to make leaks even less likely.